content/developer/addons/addon-info.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Addon Information
+tags:
+- Developers
+- Addons
+- Configuration
+- about.php
+- addon.json
+category: addons
+menu:
+  developer:
+    weight: 10
+    parent: addons
+versioning:
+  added: 2.5
+---
+
+Every Vanilla addons must declare certain information about itself. This includes:
+
+- Addon Metadata
+- Author Metadata
+- Addon support links
+- Addon configuration details
+- Details about how to load the addon
+
+## The `addon.json` file
+
+In an effort to simplify the management and creation of plugins, applications, and themes, Vanilla `2.5` and onwards uses a single unified addon information format. The new format is a `json` file that should live in the root of your addon's directory. It looks like this:
+
+### `addon.json` example
+
+```json
+{
+    "key": "example-addon",
+    "name": "Example Addon",
+    "description": "Provides an example of an Addon",
+    "version": "2.0.0",
+    "documentationUrl": "http://site.com/link-to-documentation",
+    "type": "addon",
+    "icon": "icon-location.png",
+    "mobileFriendly": true,
+    "hasLocale": false,
+    "settingsUrl": "/settings/myaddon",
+    "settingsPermission": "Garden.Settings.Manage",
+    "registerPermissions": {
+        "Plugins.MyPlugin.MyPermission": "Garden.Settings.Manage"
+    },
+    "authors": [
+        {
+            "name": "Todd Burry",
+            "email": "todd@vanillaforums.com",
+            "homepage": "https://open.vanillaforums.com/profile/todd"
+        }
+    ],
+    "require": {
+        "vanilla": ">=2.4"
+    }
+}
+```
+
+### Theme `addon.json` example
+
+Theme's tend to be a little bit simpler and would look like this:
+
+```json
+{
+    "key": "example-theme",
+    "name": "Example Theme",
+    "description": "Custom Theme Example",
+    "version": "2.0.0",
+    "type": "theme",
+    "license": "MIT",
+    "author": [
+        {
+            "name": "Adam Charron",
+            "email": "adam.c@vanillaforums.com"
+        }
+    ],
+    "layout": {
```

Comment:
I would add the a sample "sites" node, because it comes up frequently as a problem with clients.

Reply:
I've added it on both